tests/ClientTest.php

```diff
@@ -424,6 +424,8 @@ public function testAttachStacktrace(): void
 
                 $this->assertInstanceOf(Stacktrace::class, $result);
                 $this->assertNotEmpty($result->getFrames());
+                $this->assertTrue($result->getFrames()[0]->isInApp());
```

Comment:
Can we save the result of the call to `getFrames()` into a variable that we reuse later for the other assertions? This way we avoid calling that method multiple times